app/presenters/bidding_status_presenter/available.rb

```diff
@@ -19,6 +19,26 @@ def label
     'Open'
   end
 
+  def bid_label(user)
+    bid = highlighted_bid(user)
+    amount = Currency.new(bid.amount)
+
+    if bid.is_a?(NullBid)
+      I18n.t('bidding_status.available.bid_label.no_bids')
+    elsif auction.type == 'sealed_bid'
+      I18n.t('bidding_status.available.bid_label.sealed_bid',
```

Comment:
I thought about that, but the Rules classes don't really have any `presenter` style logic in them. To be honest, I don't know if it would make more sense to have `SealedBidAuctionPresenter` and `ReverseAuctionPresenter` classes?